debian/dist/Debian/rules

```diff
@@ -216,7 +220,9 @@ binary-indep: build install
 	dh_installexamples -i
 	dh_installmenu -i
 	dh_installlogrotate -i
+	dh_systemd_enable -i support-files/mariadb.service support-files/mariadb.socket
 	dh_installinit -i
+	dh_systemd_start -i --restart-after-upgrade mariadb.socket
```

Comment:
not sure. I'm assuming the abstraction is on the debian end. Just following the rules: https://wiki.debian.org/systemd/Packaging